specification/app/resource-manager/Microsoft.App/stable/2022-03-01/ManagedEnvironments.json

```diff
@@ -688,7 +688,7 @@
         },
         "runtimeSubnetId": {
           "type": "string",
-          "description": "Resource ID of a subnet that Container App containers are injected into. This subnet must be in the same VNET as the subnet defined in infrastructureSubnetId. Must not overlap with any other provided IP ranges.",
+          "description": "This field is deprecated and not used. If you wish to provide your own subnet that Container App containers are injected into, then you should leverage the infrastructureSubnetId.",
```

Comment:
>description [](http://example.com/codeflow?start=11&length=11)

Have we fixed this already for later API versions?

Reply:
Done. Here is the PR [release-app-Microsoft.App-2022-11-01-preview](https://github.com/Azure/azure-rest-api-specs/tree/release-app-Microsoft.App-2022-11-01-preview) branch:

https://github.com/Azure/azure-rest-api-specs/pull/21520

Reply:
@navba-MSFT - I would recommend closing this PR and make the change to **all versions** in the new PR you have created: https://github.com/Azure/azure-rest-api-specs/pull/21520